images/trustx-cml-initramfs.bbappend

```diff
@@ -0,0 +1,4 @@
+cleanup_boot () {
+        rm -f ${IMAGE_ROOTFS}/boot/*
```

Comment:
why is there a force needed?

Reply:
because in case in case of toradex colibri the folder was already empty, thus rm without "-f" caused "file not found" type of error

Reply:
hmm OK than it would be better to fix this in meta-trustx.